rustywind-core/src/sorter.rs

```diff
@@ -31,53 +31,51 @@ pub struct Options {
     pub sorter: Sorter,
     pub allow_duplicates: bool,
 }
+impl Options {
+    /// Extracts the [Regex] pattern that will be used for matching classes
+    pub fn extract_regex(&self) -> &Regex {
+        match &self.regex {
+            FinderRegex::DefaultRegex => &RE,
+            FinderRegex::CustomRegex(regex) => regex,
+        }
+    }
+    /// Extracts the [HashMap] that will be used for sorting classes
+    pub fn extract_sorter(&self) -> &HashMap<String, usize> {
```

Comment:
```suggestion
    pub fn sorter(&self) -> &HashMap<String, usize> {
```

Reply:
I thought it might be confusing to have 2 values and methods with the same name, hence I added the prefix. You think it's ok as just `sorter()` and `regex()` though?

Reply:
What do you think of doing something like this instead actually, where the enums themselves handle the extraction:

```rust
impl FinderRegex {
    fn get(&self) -> &Regex {
        match &self {
            Self::DefaultRegex => &RE,
            Self::CustomRegex(re) => re,
        }
    }
}
```

Reply:
ya i think i like that better